src/components/Loading.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactLoading from "react-loading";
+
+export default function Loading() {
+    return (
+        <section>
+            <h2>Loading...</h2>
+            <ReactLoading type="spokes" color="#0000FF"
+                height={200} width={100} />
+        </section>
+    )
+}
```

Comment:
Great to see the inclusion of a reusable loading component with a visual indicator during data fetching.